arm-redis/2015-08-01/swagger/redis.json

```diff
@@ -423,12 +456,8 @@
       },
       "description": "Redis cache access keys."
     },
-    "RedisReadablePropertiesWithAccessKey": {
+    "RedisReadableProperties": {
```

Comment:
@alfantp - This will be a breaking change for node sdk and xplatcli commands need to be updated.